client/state/gsuite-users/schema.js

```diff
@@ -5,34 +5,36 @@ export const usersSchema = {
 			{
 				type: 'object',
 				required: [
-					'email',
-					'mailbox',
+					'agreed_to_terms',
 					'domain',
+					'email',
 					'firstname',
-					'lastname',
 					'fullname',
+					'is_admin',
+					'lastname',
+					'mailbox',
 					'site_id',
 					'suspended',
-					'agreed_to_terms',
 				],
 				properties: {
-					mailbox: { type: 'string' },
+					agreed_to_terms: { type: 'boolean' },
 					domain: { type: 'string' },
 					email: { type: 'string' },
 					firstname: { type: 'string' },
-					lastname: { type: 'string' },
 					fullname: { type: 'string' },
+					is_admin: { type: 'boolean' },
+					lastname: { type: 'string' },
+					mailbox: { type: 'string' },
 					site_id: { type: 'integer' },
 					suspended: { type: 'boolean' },
-					agreed_to_terms: { type: 'boolean' },
```

Comment:
I've only added the new `is_admin` property in this schema, the rest of the changes is just ordering properties alphabetically so they are easier to read.

Reply:
Thanks for surfacing this fact. It made things easier.